src/olympia/addons/views.py

```diff
@@ -379,7 +378,6 @@ def license_redirect(request, version):
     return redirect(version.license_url(), permanent=True)
 
 
-@session_csrf.anonymous_csrf_exempt
```

Comment:
This may need to be reintroduced, got removed as part of the `session_csrf` removal. Totally forgot about it.